doc/glossary.rst

```diff
@@ -326,6 +326,15 @@
                 <https://ceph.io/assets/pdfs/CawthonKeyValueStore.pdf>`_ (17
                 pages).
 
+        OpenStack Swift
+                In the context of Ceph, OpenStack Swift is one of the two
+                storage interfaces supported by the Ceph Object Store. The
```

Comment:
This isn't wrong, but I might s/storage interfaces/APIs/

Reply:
I think I like that better.